internal/driver/deploy/releases/linode-blockstorage-csi-driver.yaml

```diff
@@ -471,7 +462,7 @@ spec:
             secretKeyRef:
               key: token
               name: linode
-        image: linode/linode-blockstorage-csi-driver:latest
+        image: linode/linode-blockstorage-csi-driver:version-upgrade
```

Comment:
Should change this back to latest right?

Reply:
good catch!